core/primitives/address.cpp

```diff
@@ -42,6 +51,46 @@ namespace fc::primitives {
         [](const BLSPublicKeyHash &v) { return Protocol::BLS; });
   }
 
+  fc::outcome::result<bool> Address::verifySyntax(
+      const gsl::span<uint8_t> &seed_data) const {
+    return visit_in_place(
+        data,
+        [](uint64_t v) { return true; },
+        [&seed_data](
+            const Secp256k1PublicKeyHash &v) -> fc::outcome::result<bool> {
+          if (seed_data.size() != std::tuple_size<Sec256k1PublicKey>::value)
```

Comment:
I wish there is a constant in Sec256k1 defining the length of public key.

Reply:
I defined one where I needed it: https://github.com/filecoin-project/cpp-filecoin/blob/master/core/primitives/address_codec.cpp#L17, I assume this can be generalized for the entire application

Reply:
Not a hash length, but length of public key itself.

Reply:
By the way I moved length hash of blake2b-160 to https://github.com/filecoin-project/cpp-filecoin/blob/master/core/crypto/blake2/blake2b160.hpp#L17